cpp/src/utilities/cython.cpp

```diff
@@ -185,15 +243,15 @@ weight_t call_louvain(raft::handle_t const& handle,
   if (graph_container.graph_ptr_type == graphTypeEnum::GraphCSRViewFloat) {
     std::pair<size_t, float> results =
       louvain(handle,
-              *(graph_container.graph_ptr_union.GraphCSRViewFloatPtr),
+              *(graph_container.graph_ptr_union.GraphCSRViewFloatPtr.get()),
               reinterpret_cast<int*>(parts),
               max_level,
               static_cast<float>(resolution));
     final_modularity = results.second;
   } else {
     std::pair<size_t, double> results =
       louvain(handle,
-              *(graph_container.graph_ptr_union.GraphCSRViewDoublePtr),
+              *(graph_container.graph_ptr_union.GraphCSRViewDoublePtr.get()),
```

Comment:
Do you need the .get() here?  Doesn't the * operator do the right thing?

Reply:
ah, thanks, I don't need `.get()`.